src/qt/pivx/sendchangeaddressdialog.cpp

```diff
@@ -70,6 +72,25 @@ void SendChangeAddressDialog::reset()
     close();
 }
 
-SendChangeAddressDialog::~SendChangeAddressDialog(){
+void SendChangeAddressDialog::save()
+{
+    // validate address
+    if (!walletModel->validateAddress(ui->lineEditAddress->text())) {
+        inform(tr("Invalid change address"));
```

Comment:
Would just put "Invalid address", the user is already in the change address dialog which has only a single edit box.

Reply:
done.